app/messaging/process-return-message.js

```diff
@@ -1,6 +1,9 @@
+const { processSubmittedSettlement } = require('../inbound')
 const processReturnMessage = async (message, receiver) => {
   try {
     await receiver.completeMessage(message)
+    const settlement = message.body
+    await processSubmittedSettlement(settlement)
```

Comment:
couple things on this:

1. wrong order: as part of our message pattern we `completeMessage` right at the end becuz if `processSubmittedSettlement` were to fail, the message would have been marked as complete and never re-sent so we'd loose data
2. log: sommit that we do here is log a message sim to `console.log('Payment request submitted:', util.inspect(paymentRequest, false, null, true))`
3. name: I would suggest we rename line 6 function to `processReturnSettlement`, again, in line with sim. funcs

Reply:
actioned as above